contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/SubmissionCallbackServlet.java

```diff
@@ -144,6 +155,21 @@ protected void doPostJSON(HttpServletRequest request, HttpServletResponse respon
         return;
     }
 	
+	public Map<String,Object> obtainGlobalTurnitinLTITool(String turnitinSite){
+		M_log.debug("Getting global TII LTI tool");
```

Comment:
Besides what I've said on the other comments, there was also a problem injecting/accessing the LTIService from a method outside the servlet.